test/tests/mesh/periodic_node_map/tests

```diff
@@ -8,6 +8,7 @@
 
     requirement = "The system shall produce a valid periodic node map with only one periodic constraint direction."
     recover = false  # Steady Test (Needs to be here since we don't want the checkpoints clobbering each other)
+    mesh_mode = 'replicated' # We no longer ghost all periodic boundaries for all procs
```

Comment:
mesh_mode = 'replicated distributed'? I want to test both

Reply:
These tests use a custom object that expects there to be a certain number of entries in their periodic node map. We no longer ghost all boundaries just for grins, so we no longer have all the boundary nodes when we are running distributed

Reply:
Nice!